srcpkgs/plymouth/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'plymouth'
 pkgname=plymouth
-version=0.9.4
-revision=2
+version=0.9.5
+revision=1
 build_style=gnu-configure
 configure_args="--with-system-root-install=no \
  --without-rhgb-compat-link --enable-systemd-integration=no \
- --enable-gdm-transition $(vopt_enable gtk3 gtk) $(vopt_enable pango) \
```

Comment:
outdated option or did it have to be disabled?

Reply:
Just outdated, it has been removed from the codebase.